SCClassLibrary/Common/Control/Server.sc

```diff
@@ -222,6 +225,9 @@ ServerOptions {
 		if (maxLogins.notNil, {
 			o = o ++ " -l " ++ maxLogins;
 		});
+		if (safetyClipThreshold.notNil, {
+			o = o ++ " -s " ++ safetyClipThreshold;
```

Comment:
I think it would be good to have a platform switch here (just for adding the `-s` option)